lib/manual_and_sections_redirecter.rb

```diff
@@ -0,0 +1,67 @@
+class ManualAndSectionsRedirecter
+  def initialize(args)
+    @publishing_api = SpecialistPublisher.services(:publishing_api)
+    @logger = args[:logger] || STDOUT
```

Comment:
Is the logger injected solely to clean up the output when running tests. I think there's a more RSpec way of doing this which is to wrap calls to this class in `expect { ... }.to output.to_stdout`. Just a thought.